backend/Modules/Identity/HitMeApp.Indentity/Infrastructure/Exceptions/IdentityModuleExceptionMapper.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Net;
+using HitMeApp.Indentity.Application.Exceptions;
+using HitMeApp.Indentity.Core.Exceptions;
+using HitMeApp.Shared.DDD;
+using HitMeApp.Shared.Infrastructure.Exceptions;
+
+namespace HitMeApp.Indentity.Infrastructure.Exceptions
+{
+    internal sealed class IdentityModuleExceptionMapper : IExceptionToResponseMapper
+    {
+        public ExceptionResponse Map(Exception exception)
+            => exception switch
+            {
+                UserAlreadyExistsException ex => new ExceptionResponse(ex.Message, HttpStatusCode.Conflict),
+                DomainException ex => new ExceptionResponse(ex.Message),
+                AppException ex => new ExceptionResponse(ex.Message),
+                Exception => new ExceptionResponse("Unknown error occured in the users module", HttpStatusCode.InternalServerError),
```

Comment:
This is `Identity` module now. Maybe we could use `nameof` here to avoid such problems in the future?

Reply:
Definitely. String literals cut deep in scenarios like this one.